tools/generate-launcher

```diff
@@ -20,7 +20,7 @@ PYTHON2_CMD=${5:-python2}
 #  exit 1
 #fi
 
-EPREFIX=`$PYTHON2_CMD -c 'import os,sys;print os.path.realpath(sys.argv[1])' $EPREFIX`
+EPREFIX=`$PYTHON2_CMD -c 'import os,sys;print(os.path.realpath(sys.argv[1]))' $EPREFIX`
```

Comment:
We aren't dropping python2 support, so maybe do `sys.stdout.write` instead to retain compatability for both?